torchvision/transforms/functional.py

```diff
@@ -20,18 +21,25 @@
 from . import functional_tensor as F_t
 
 
-def _is_pil_image(img):
-    if accimage is not None:
-        return isinstance(img, (Image.Image, accimage.Image))
-    else:
-        return isinstance(img, Image.Image)
+_is_pil_image = F_pil._is_pil_image
+
+
+def _get_image_size(img: Tensor) -> List[int]:
+    """Returns image sizea as (w, h)
+    """
+    if isinstance(img, torch.Tensor):
+        return F_t._get_image_size(img)
 
+    return F_pil._get_image_size(img)
 
-def _is_numpy(img):
+
+@torch.jit.ignore
```

Comment:
I think `torch.jit.unused` is better here, otherwise those functions won't be able to be serialized / exported